packages/integration-sdk-testing/src/jest.ts

```diff
@@ -222,12 +226,68 @@ export function toMatchGraphObjectSchema<T extends Entity>(
     };
   }
 
+  return toMatchSchema(received, newEntitySchema);
+}
+
+export interface ToMatchRelationshipSchemaParams {
+  /**
+   * The schema that should be used to validate the input data against
+   */
+  schema?: GraphObjectSchema;
+}
+
+export function toMatchDirectRelationshipSchema<T extends ExplicitRelationship>(
+  received: T | T[],
+  params?: ToMatchRelationshipSchemaParams,
+) {
+  const dataModelIntegrationSchema = dataModel.IntegrationSchema;
+  const schema = params?.schema || {};
+
+  const graphObjectSchemas = collectSchemasFromRef(
+    dataModelIntegrationSchema,
+    '#GraphObject',
+  );
+
+  const directRelationshipSchema = {
+    $schema: 'http://json-schema.org/draft-07/schema#',
+    $id: '#DirectRelationship',
+    description:
+      'An edge in the graph database that represents a Relationship. This reference schema defines common shared properties among most Entities.',
+    type: 'object',
+    allOf: [
+      {
+        $ref: '#GraphObject',
+      },
+      {
+        additionalProperties: {
+          type: ['boolean', 'integer', 'null', 'number', 'string'],
+        },
+      },
+    ],
+  };
```

Comment:
There is no `#Relationship` schema in the data model, but this defines what it _would_ look like.

The tricky part will be figuring how to share logic between `ExplicitRelationship` and `MappedRelationship`. Not sure if we will need one JSON schema for each or what.